programs/drift/src/validation/order.rs

```diff
@@ -43,12 +43,6 @@ pub fn validate_order(
 fn validate_market_order(order: &Order, step_size: u64, min_order_size: u64) -> DriftResult {
     validate_base_asset_amount(order, step_size, min_order_size, order.reduce_only)?;
 
-    validate!(
-        order.auction_start_price > 0 && order.auction_end_price > 0,
-        ErrorCode::InvalidOrderAuction,
-        "Auction start and end price must be greater than 0"
-    )?;
-
```

Comment:
was this just a waste? probably would be good to add test if they were negative then

Reply:
nah ill add em back